plugin/src/test/resources/io/jenkins/plugins/casc/plugins/ProxyConfigTest.yml

```diff
@@ -1,8 +0,0 @@
-plugins:
-  proxy:
```

Comment:
thoughts on whether this is still needed?

Reply:
It properly is 😅